### PR TITLE
XZ dep for libunwind

### PR DESCRIPTION
**Pull request by boegel**
for https://github.com/hpcugent/easybuild-easyconfigs/pull/2892

### CODE REVIEW

File: easybuild/easyconfigs/l/libunwind/libunwind-1.1-foss-2016a.eb [hunk 1]
```diff
@@ -19,6 +19,12 @@ checksums = [
     'fb4ea2f6fbbe45bf032cd36e586883ce',     # libunwind-1.1.tar.gz
 ]
 
+dependencies = [
+    ('XZ', '5.2.2'),
+]
+
+preconfigopts = 'export LIBS="$LIBS -llzma" && '
```
Comment (boegel) [inline easybuild/easyconfigs/l/libunwind/libunwind-1.1-foss-2016a.eb:26]: worth mentioning: easier workaround than patching the broken Makefile, cfr. https://bugs.gentoo.org/show_bug.cgi?id=444050

**Comment by geimer** [comment]
Weird that this didn't show up as a problem during my testing...  It works w/o XZ on my system...
Anyway, thanks @boegel

**Comment by boegel** [comment]
@geimer: I think the problem only pops up when you _do_ have XZ/`liblzma.*` on your system, due to the bug in the `Makefile`...

**Comment by geimer** [comment]
@boegel: I _do_ have it on my system, but `configure` says

```
checking whether to support LZMA-compressed symbol tables... auto
checking for lzma_mf_is_supported in -llzma... no
```

And after that no further mentioning of `lzma`...

**Comment by boegel** [comment]
Version too old (must be ancient then)?

**Pull request merged**